charts/internal/machine-controller-manager/seed/values.yaml

```diff
@@ -30,7 +30,13 @@ resources:
       memory: 70Mi
     limits:
       memory: 1024Mi
+    maxAllowed:
+      cpu: 2
+      memory: 5G
   mcmProviderAlicloud:
```

Comment:
what's the relationship between limits and maxallowed?  value of limits is set in pod as resource limitation.

Reply:
Actually, maxAllowed and minAllowed, defines a range to cap resource `request` recommendations, while the `limit` limits the ussage of the resource. The reason for adding maxAllowed is to enshure that pod is schedulable, as scheduling is base on requests.